content/c-sharp/concepts/comments/comments.md

```diff
@@ -0,0 +1,86 @@
+---
+Title: 'C# Comments'
+Description: 'A comment is a piece of text within a program that is not executed.'
+Subjects: 
+  - 'Computer Science'
+  - 'Code Foundations'  
+Tags: 
+  - 'Syntax'
+  - 'Comments'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+A **comment** is a piece of text within a program that is not executed. It can be used to provide additional information to aid in understanding the code.
+
+
+### Single-line Comments
+
+#### Syntax
+
+Single-line comments are created using two consecutive forward slashes. The compiler ignores any text after `//` on the same line.
+
+```cs
+// Comment goes here
+executing code // Comment goes here
+```
+
+#### Example
+
+The following example uses a single-line comment before a line of code:
+
+```cs
+// This line will denote a comment in C-sharp
+Console.WriteLine("Hello World!");
+```
+
+The following example uses a single-line comment after a line of code:
+
+```cs
+Console.WriteLine("Hello World!"); // This is a comment
+```
+
+### Multi-line Comments
+
+Multi-line comments are created using `/* to begin the comment, and */` to end the comment. The compiler ignores any text in between.
+
+```cs 
+/*
+This is all commented out.
+None of it is going to run!
+*/
+```
+
+### XML Comments
+
+XML comments are structured comments that **produce API documentation**. The C# compiler produces an XML file that contains structured data representing the comments. Other tools can process that XML output to create human-readable documentation in the form of web pages or PDF files, for example.
+API docs, or API description documents, are the collection of references, tutorials, and examples that help developers use your API.
```

Comment:
```suggestion
```

I think this information, as currently written, makes more sense in the existing [API entry](https://www.codecademy.com/resources/docs/general/api) under the General topic.